### PR TITLE
Add cvt_cvt-w24-384-in22k_finetuned-in1k_4

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    preprocessing = functools.partial(load_preprocess_images, processor=processor, image_size=image_size)
+    wrapper = PytorchWrapper(identifier=name, model=model, preprocessing=preprocessing)
+    wrapper.image_size = image_size
+
+    return wrapper
+
+
+def get_layers(name):
+    assert name == 'cvt_cvt-w24-384-in22k_finetuned-in1k_4'
+    # layers = []
```
Comment (mike-ferguson) [inline brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py:36]: Lines  37-40 can be uncommented out, and line 41 can be deleted. This must have been from an old version of that model that I was forcing it to use a certain layer. We can use all layers now.

File: brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+from PIL import Image
+import numpy as np
+import torch
+
+"""
+Template module for a base model submission to brain-score
+"""
+
+
+def get_model_list(name):
```
Comment (mike-ferguson) [inline brainscore_vision/models/cvt_cvt-w24-384-in22k_finetuned-in1k_4/model.py:14]: This method can be deleted (lines 14-18)

**Review by mike-ferguson — CHANGES_REQUESTED** [review]
lgtm once the comments are addressed

**Review by mike-ferguson — APPROVED** [review]

File: brainscore_vision/models/cvt_cvt_w24_384_in22k_finetuned_in1k_4/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+numpy
+torch
+transformers==4.19.0
+PIL
```
Comment (deirdre-k) [inline brainscore_vision/models/cvt_cvt_w24_384_in22k_finetuned_in1k_4/requirements.txt:4]: @pellegreene PIL is no longer supported for Python 3 and instead has been replaced by [Pillow](https://pillow.readthedocs.io/en/stable/), same as https://github.com/brain-score/vision/pull/946

**Comment by mike-ferguson** [comment]
god merged as Jenkins has been running (>12 hours)

**Pull request merged**